internal/datastore/crdb/migrations/zz_migration.0007_add_integrity_columns.go

```diff
@@ -0,0 +1,29 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v5"
+)
+
+const (
+	addIntegrityColumnsQuery = `
+		ALTER TABLE relation_tuple
+		ADD COLUMN IF NOT EXISTS integrity_hash BYTEA DEFAULT NULL,
+		ADD COLUMN IF NOT EXISTS integrity_key_id VARCHAR(255) DEFAULT NULL;
+	`
+)
+
+func init() {
+	err := CRDBMigrations.Register("add-integrity-columns", "add-relationship-counters-table", addIntegrityColumns, noAtomicMigration)
```

Comment:
is there a must register? i swear it's been written at some point

Reply:
Nope, we never added it. We could, but this seems fine